artifacts/api-manager-artifacts/pattern-2/wso2am-p2-mysql-dbscripts.yaml

```diff
@@ -0,0 +1,2491 @@
+#Copyright (c)  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: wso2am-p2-mysql-dbscripts
+  namespace: wso2-system
+data:
+  init.sql: |
+    DROP DATABASE IF EXISTS DASHBOARD_DB;
+    DROP DATABASE IF EXISTS APIM_ANALYTICS_DB;
+    DROP DATABASE IF EXISTS PERMISSION_DB;
+    DROP DATABASE IF EXISTS GEO_LOCATION_DATA;
+    DROP DATABASE IF EXISTS BUSINESS_RULES_DB;
+    DROP DATABASE IF EXISTS WSO2AM_SHARED_DB;
+    DROP DATABASE IF EXISTS WSO2AM_APIMGT_DB;
+    DROP DATABASE IF EXISTS WSO2_CLUSTER_DB;
+    DROP DATABASE IF EXISTS PERSISTENCE_DB;
+
+    CREATE DATABASE DASHBOARD_DB;
+    CREATE DATABASE APIM_ANALYTICS_DB;
+    CREATE DATABASE PERMISSION_DB;
+    CREATE DATABASE GEO_LOCATION_DATA;
+    CREATE DATABASE BUSINESS_RULES_DB;
+    CREATE DATABASE WSO2AM_SHARED_DB;
+    CREATE DATABASE WSO2AM_APIMGT_DB;
+    CREATE DATABASE WSO2_CLUSTER_DB;
+    CREATE DATABASE PERSISTENCE_DB;
+
+    CREATE USER IF NOT EXISTS 'wso2carbon'@'%' IDENTIFIED BY 'wso2carbon';
+
+    GRANT ALL PRIVILEGES ON DASHBOARD_DB.* TO 'wso2carbon'@'%' IDENTIFIED BY 'wso2carbon';
+    GRANT ALL PRIVILEGES ON WSO2AM_APIMGT_DB.* TO 'wso2carbon'@'%' IDENTIFIED BY 'wso2carbon';
+    GRANT ALL PRIVILEGES ON APIM_ANALYTICS_DB.* TO 'wso2carbon'@'%' IDENTIFIED BY 'wso2carbon';
+    GRANT ALL PRIVILEGES ON PERMISSION_DB.* TO 'wso2carbon'@'%' IDENTIFIED BY 'wso2carbon';
+    GRANT ALL PRIVILEGES ON GEO_LOCATION_DATA.* TO 'wso2carbon'@'%' IDENTIFIED BY 'wso2carbon';
+    GRANT ALL PRIVILEGES ON BUSINESS_RULES_DB.* TO 'wso2carbon'@'%' IDENTIFIED BY 'wso2carbon';
+    GRANT ALL PRIVILEGES ON WSO2AM_SHARED_DB.* TO 'wso2carbon'@'%' IDENTIFIED BY 'wso2carbon';
+    GRANT ALL PRIVILEGES ON WSO2_CLUSTER_DB.* TO 'wso2carbon'@'%' IDENTIFIED BY 'wso2carbon';
+    GRANT ALL PRIVILEGES ON PERSISTENCE_DB.* TO 'wso2carbon'@'%' IDENTIFIED BY 'wso2carbon';
+
+    USE APIM_ANALYTICS_DB;
+
+    CREATE TABLE IF NOT EXISTS AM_USAGE_UPLOADED_FILES (
+    FILE_NAME varchar(255) NOT NULL,
+    FILE_TIMESTAMP TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    FILE_PROCESSED tinyint(1) DEFAULT 0,
+    FILE_CONTENT MEDIUMBLOB DEFAULT NULL,
+    PRIMARY KEY (FILE_NAME, FILE_TIMESTAMP)
+    );
```

Comment:
Shall we indent statements similar to this?
eg:
```sql
    CREATE TABLE IF NOT EXISTS AM_USAGE_UPLOADED_FILES (
        FILE_NAME varchar(255) NOT NULL,
        FILE_TIMESTAMP TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
        FILE_PROCESSED tinyint(1) DEFAULT 0,
        FILE_CONTENT MEDIUMBLOB DEFAULT NULL,
        PRIMARY KEY (FILE_NAME, FILE_TIMESTAMP)
    );
```

Reply:
addressed via https://github.com/wso2/k8s-wso2am-operator/pull/89/commits/8b163758448b29fec66bc8139ad184f405bf1f28